.gitignore

```diff
@@ -131,3 +131,5 @@ dmypy.json
 # Pyre type checker
 .pyre/
 
+harmonica/tmp/
```

Comment:
I'd prefer to just have this in output/tmp. `output/` is already set to ignore anything in there with the local .gitignore in that folder. But this is fine if you prefer it this way.